docs/howtos/pyenv_virtualenv.md

```diff
@@ -0,0 +1,62 @@
+### Virtual Environments
+
+Virtual environments allow a developer to have an encapsulated Python environment, using a specific version of Python, and with dependencies installed in a way that only affect the virtual environment. This is important as different projects or even different versions of the same project may have different dependencies, and virtual environments allow you to switch between them seamlessly and explicitly.
+
+### Using `pyenv virtualenv` with pyenv
+
+To create a virtualenv for the Python version used with pyenv, run `pyenv virtualenv`, specifying the Python version you want and the name of the virtualenv directory. For example, because we can make a virtual environment for Kolibri using Python 3.9.9:
+
+```sh
+$ pyenv virtualenv 3.9.9 kolibri-py3.9
+```
+
+*If you get 'command not found' or a similar error, and pyenv virtualenv is not installed, [please follow the installation instructions](https://github.com/pyenv/pyenv-virtualenv#installation).*
```

Comment:
Not sure if it's another copy-paste from `md` mishap, but this link is not displayed properly:
![Selection_015](https://user-images.githubusercontent.com/1457929/183341207-023478f8-cb3e-4354-a176-5fb0af75d75b.png)

Reply:
I think this is actually a copy paste error from RST into MD.